src/main/res/values/strings.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
-    <string name="app_name">AnySoftKeyboard - change_me Language Pack</string>
-    <string name="keyboard_name">change_me keyboard</string>
-    <string name="dictionary_name">change_me</string>
+    <string name="app_name">AnySoftKeyboard - Icelandic Language Pack</string>
+    <string name="keyboard_name">Icelandic keyboard</string>
```

Comment:
Change to
`<string name="keyboard_name">Icelandic</string>`
because it is displayed on the keyboard toggle button and would be too small otherwise.
(We should change it in the template...)